client-v2/src/components/Clipboard.tsx

```diff
@@ -192,57 +199,64 @@ class Clipboard extends React.Component<ClipboardProps> {
+                          {...afProps}
                         >
-                          {(supporting, sProps, i, arr) => (
-                            <>
-                              {i < arr.length ? <SupportingDivider /> : null}
-                              <CollectionItem
-                                uuid={supporting.uuid}
-                                frontId={clipboardId}
-                                parentId={articleFragment.uuid}
-                                getNodeProps={() => sProps}
-                                size="small"
-                                displayType="polaroid"
-                                onSelect={id =>
-                                  this.props.selectArticleFragment(id, true)
-                                }
-                                onDelete={() =>
-                                  this.props.removeSupportingCollectionItem(
-                                    articleFragment.uuid,
-                                    supporting.uuid
-                                  )
-                                }
-                              />
-                            </>
-                          )}
-                        </ArticleFragmentLevel>
-                      </CollectionItem>
-                    </ArticleWrapper>
+                          <ArticleFragmentLevel
+                            articleFragmentId={articleFragment.uuid}
+                            onMove={this.handleMove}
+                            onDrop={this.handleInsert}
+                            displayType="polaroid"
+                          >
+                            {(supporting, sProps, i, arr) => (
+                              <>
+                                {i < arr.length ? <SupportingDivider /> : null}
+                                <CollectionItem
+                                  uuid={supporting.uuid}
+                                  frontId={clipboardId}
+                                  parentId={articleFragment.uuid}
+                                  getNodeProps={() => sProps}
+                                  size="small"
+                                  displayType="polaroid"
+                                  onSelect={id =>
+                                    this.props.selectArticleFragment(id, true)
+                                  }
+                                  onDelete={() =>
+                                    this.props.removeSupportingCollectionItem(
+                                      articleFragment.uuid,
+                                      supporting.uuid
+                                    )
+                                  }
+                                />
+                              </>
+                            )}
+                          </ArticleFragmentLevel>
+                        </CollectionItem>
+                      </ArticleWrapper>
+                      <FullDivider />
```

Comment:
The addition of `<FullDivider />` is the only change here, the rest is just indentation..